images/base/files/usr/local/bin/entrypoint

```diff
@@ -23,6 +23,19 @@ configure_containerd() {
   if [[ "$(stat -f -c %T /kind)" == 'zfs' ]]; then
     sed -i 's/snapshotter = "overlayfs"/snapshotter = "native"/' /etc/containerd/config.toml
   fi
+  if [[ "${KIND_EXPERIMENTAL_STARGZ:-}" == "1" ]] ; then
```

Comment:
For extensibility, the env var name should be like KIND_CONTAINERD_SNAPSHOTTER=stargz

Reply:
Fixed it.